303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/handlers/pricing/NativePostgresqlPricingRulesHandlerIT.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.qe.vertx.sql.handlers.pricing;
+
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import io.quarkus.qe.vertx.sql.test.profiles.NativeProfile;
+import io.quarkus.test.junit.NativeImageTest;
+
+@NativeImageTest
+@EnabledIfSystemProperty(named = NativeProfile.QUARKUS_PROFILE, matches = NativeProfile.NATIVE)
```

Comment:
With `NativeImageTest` annotation, this other precondition is not required: `@EnabledIfSystemProperty(named = NativeProfile.QUARKUS_PROFILE, matches = NativeProfile.NATIVE)`